libraries/AP_HAL_ChibiOS/hwdef/ReaperF745v2/hwdef.dat

```diff
@@ -118,17 +118,20 @@ STORAGE_FLASH_PAGE 1
 # enable logging to dataflash
 define HAL_LOGGING_DATAFLASH_ENABLED 1
 
-# spi devices
-SPIDEV bmi270    SPI3 DEVID1 BMI270_CS  MODE3  1*MHZ  4*MHZ
+# spi devices bmi270(v2) icm42688 (v3)
+SPIDEV bmi270    SPI3 DEVID1 GYRO_CS  MODE3  1*MHZ  4*MHZ
+SPIDEV icm42688  SPI3 DEVID1 GYRO_CS  MODE3  2*MHZ  8*MHZ
```

Comment:
On all the other ports of this nature I have done it needs to be DEVID2
Also you should be able to go up to 16Mhz for the 42688

Reply:
its not necessary if they cant exist at the same time AIFAIK  ... the MAtekH743 does not require it

Reply:
every other instantiation is 8Mhz, except the MatekH743...dont know if H7 makes a difference or not

Reply:
> its not necessary if they cant exist at the same time AIFAIK ... the MAtekH743 does not require it

Just saying what was required on other boards ...

Reply:
> every other instantiation is 8Mhz including the ones you just did, except the MatekH743...dont know if H7 makes a difference or not

Nora is 16Mhz. Not sure why you are pushing back against this Henry? Faster is better if the board supports it. Depending on the clock the actual selected speed will be somewhat lower. Please try 16Mhz.

Reply:
I am just going by previous patterns....not knowing anything else....including when you did the MambaF405 with this IMU..you set it at 8MHz not 16MHz..the only hwdefs which set it at 16MHZ were H7 processors...no F4 boards have it set....just previous patterns, not knowledge....happy to change to 16MHZ if you are sure its better....but then we should update all the previous boards using it at 8MHZ, correct?

Reply:
DEVID1 is fine, as on the same CS pin, so can't have both at once